usmqe_tests/api/others/test_user.py

```diff
@@ -200,6 +200,80 @@ def test_user_change_password_to_invalid(valid_new_normal_user, invalid_password
         test.edit_user(valid_new_normal_user["username"], edit_back_data)
 
 
+@pytest.mark.xfail
+@pytest.mark.negative
+def test_add_user_invalid_password(valid_session_credentials,
+                                   valid_normal_user_data, invalid_password):
+    """@pylatest api/user.add_delete
+    API-users: add and delete
+    *************************
+
+    .. test_metadata:: author ebondare@redhat.com
+
+    Description
+    ===========
+
+    Attempt to add a user with invalid password
+    """
+    test = tendrlapi_user.ApiUser(auth=valid_session_credentials)
+    """@pylatest
+    .. test_step:: 1 api/user.add_delete
+
+        Attempt to add user using an invalid password, either too long or too short.
+
+        Send **PUT** request to ``APIURL/users/test2`` with data from fixture
+        valid_normal_user_data where are specified keys: email, username, name, role
+
+    .. test_result:: 1
+
+        User should not be created.
+
+        Return code should be 422.
+
+        This check might fail due to https://bugzilla.redhat.com/show_bug.cgi?id=1610947
+    """
+    user_data_password_invalid = valid_normal_user_data
+    user_data_password_invalid["password"] = invalid_password
```

Comment:
Ops, here we need to use:

```
user_data_password_invalid = copy.deepcopy(valid_normal_user_data)
```
otherwise `valid_normal_user_data` will contain invalid password as well.

Since fixture `valid_normal_user_data` is recreated for each test case, it won't impact other tests. But if we use
this fixture again in this test case, we will be surprised that it doesn't contain expected data.